### PR TITLE
Fix profile links and dividers

### PR DESCRIPTION
**Pull request by maxious**

### CODE REVIEW

File: src/shared/SellerProfile/Body.js [hunk 1]
```diff
@@ -137,7 +137,7 @@ const Body = (props) => {
                                        rel="external">Product support</a>
                                 </p>
                             </div>
-                            <hr styleName="productHr" />
+                            {i < Object.keys(products).length-1 ? (<hr styleName="productHr" />) : null}
```
Comment (matt-sm) [inline src/shared/SellerProfile/Body.js]: could be
`i < Object.keys(products).length-1 &&`

File: src/shared/SellerProfile/Body.js [hunk 2]
```diff
@@ -108,7 +108,7 @@ const Body = (props) => {
 
             <Row title="Digital products" show={!isEmpty(products)}>
 
-            <SimpleAccordion title="Learn more" show={!isEmpty(public_profile)}>
+            <SimpleAccordion title="Learn more" show={isEmpty(public_profile) || public_profile !== false}>
```
Comment (matt-sm) [inline src/shared/SellerProfile/Body.js]: so the public profile could have a value "false"?

Comment (maxious) [inline src/shared/SellerProfile/Body.js]: yes, explicit false means admin only

File: src/shared/Card/Card.js [hunk 3]
```diff
@@ -6,36 +6,46 @@ import Badges from '../Badges';
...
-    </section>
-  </a>
-);
+const Card = ({ title, link, badges = {}, description, products = {}, services = {} }) => {
+    // calculate badges
+    var calculated_badges = new Object();
+    Object.assign(calculated_badges, badges);
+    //calculated_badges['recruitment'] = (recruiter === 'yes');
+    calculated_badges['product'] = !isEmpty(products);
+    badges = calculated_badges;
```
Comment (matt-sm) [inline src/shared/Card/Card.js]: not a fan of mutating props, perhaps better to:
`badges = Object.assign({}, product: !isEmpty(products), badges);`

File: src/bundles/SellerRegistration/components/ApplicationPreview/ApplicationPreview.js [hunk 4]
```diff
@@ -64,7 +64,7 @@ const mapStateToProps = ({ application }, { documentsUrl, onClick, ...rest }) =>
       ...body
     } = application;
 
-    let assessed = application.assessed_domains;
+    let assessed = application.assessed_domains || domains.assessed;
```
Comment (matt-sm) [inline src/bundles/SellerRegistration/components/ApplicationPreview/ApplicationPreview.js]: in keeping with our recent bugs, can domains be `undefined`?

File: src/shared/Card/Card.js [hunk 5]
```diff
@@ -6,36 +6,46 @@ import Badges from '../Badges';
...
-        <div styleName="services">
-          {Object.keys(services).map((service, i) => (
-            <span key={i}>{service}</span>
-          ))}
-        </div>
-      )}
-    </section>
-  </a>
-);
+const Card = ({ title, link, badges = {}, description, products = {}, services = {} }) => {
```
Comment (matt-sm) [inline src/shared/Card/Card.js]: I would add a unit test for Card.js

File: src/bundles/SellerRegistration/components/ApplicationPreview/ApplicationPreview.js (hunk 4 above)
Comment (maxious) [inline src/bundles/SellerRegistration/components/ApplicationPreview/ApplicationPreview.js]: fallback to [] empty array

File: src/shared/Card/Card.js (hunk 5 above)
Comment (maxious) [inline src/shared/Card/Card.js]: added

File: src/shared/Card/Card.test.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+        badges: {'sme': true},
+        description: "Result description",
+        products: {"prodA": {}},
+        services: {}
+    }
+
+    const component = mount(
+        <Card {...props} />
+    );
+    console.log( component.html());
```
Comment (matt-sm) [inline src/shared/Card/Card.test.js]: remove

Comment (maxious) [inline src/shared/Card/Card.test.js]: removed

**Pull request merged**